app/assets/javascripts/pages.js

```diff
@@ -1,3 +1,9 @@
 $(document).ready(function() {
   $(".button-collapse").sideNav();
+  $('select').material_select();
```

Comment:
Mixed double and single quotes.<br>Identifier 'material_select' is not in camel case.